### PR TITLE
New Installer with UI

### PR DESCRIPTION
**Pull request by marcvaneijk**
- Prepare the SafeOS for deployment
- Prepare the Azure Stack Development Kit Installation
- Rerun and Gather Logs
- Reboot to SafeOS

Replaces
PrepareBootFromVHD, BootMenuNoKVM and Unattend files

### CODE REVIEW

File: Deployment/BootMenuNoKVM.ps1 [hunk 1]
```diff
@@ -1,71 +0,0 @@
-﻿# Copyright (c) Microsoft Corporation. All rights reserved.
-# See LICENSE.txt in the project root for license information. 
-
-<# 
-
```
Comment (MatthewMcGlynn) [inline Deployment/BootMenuNoKVM.ps1]: Is there a reason we cannot keep these files as is to support any existing users? @knithinc - do we any dependencies on these scripts?

Comment (marcvaneijk) [inline Deployment/BootMenuNoKVM.ps1]: The new script replaces all functionalities of the current different deployment scripts and files in this folder and also provides additional capabilities. Have different script with the same purpose can be confusing for users.

**Comment by marcvaneijk** [comment]
Closing cause of fork commit issue. Resubmitting.

File: Deployment/BootMenuNoKVM.ps1 (hunk 1 above)
Comment (haralsongithub) [inline Deployment/BootMenuNoKVM.ps1]: I am getting an error when I instantiate the script to prep the cloud.vhdx

>>
The property 'Name' cannot be found on this object. Verify that the property exists and can be set.
At C:\Azure_Installer\asdk-installer.ps1:1281 char:1
+ $Runspace_Jobs_Properties.Name = "Jobs"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyAssignmentException

Initialize environment. Please wait...
>>

any thoughts?  Windows 2012R2 system

**Pull request closed without merging**